### DISCUSSION TITLE
Java Day Istanbul 2023

### DISCUSSION THREAD

**Discussion by insectengine** [quarkusio/quarkus#32871] [Events]
Java Day Istanbul is one of the most effective international community driven software conference of Turkey supported by Istanbul Java User Group.

**Date:** May 6, 2023
**Location:** Istanbul, Turkey
**Event Type:** In Person

https://www.javaday.istanbul/

### Quarkus Sessions

**Enter Serverless Functions Journey with Quarkus (Workshop)** 
**Time:** 13:00 - 16:45
**Speaker:** Daniel Oh
**Abstract:** This workshop is designed to be a hands-on experience on how developers can build agnostic serverless Java functions using Quarkus, a new Kubernetes native Java framework with familiar tools (e.g. CLI, Maven, Gradel). The workshop covers deploying service functions to AWS Lambda, optimizing the functions, making them portable across multiple serverless platforms ( e.g., AWS Lambda, Azure Function, Google Cloud Platform, Kubernetes Knative), and using the latest handy command-line tool (e.g. Kn func) to enable a Buildpack for function development & deployment in minutes. Lab participants will be provided a free sandbox for serverless deployments.  
The workshop attendees need to bring their own laptops to go through the lab. I’ll also provide the way individual developers sign in a sandbox(free) for serverless function deployment. The workshop guide (WIP) is here - http://bit.ly/quarkus-serverless-labs

**Developing Kubernetes Operators with Java Operator SDK** 
**Time:** 13:00 - 13:45 
**Speaker:** Aykut Bulgu
**Abstract:** ava developers have been using the fabric8 library for some time, but it never provided a fully-fledged SDK to develop an operator like gophers do easily via Go Operator SDK or kubebuilder today.  The Java Operator SDK comes as a savior for Java Developers. It provides a new way of automation on Kubernetes for Java developers who want to write operators in the language they love.  

Doesn’t matter if you’ve developed Kubernetes operators before or using Kubernetes currently. If you are a Java developer, come on and join this session to explore the beauty of Kubernetes and how to extend it with the language you love. 

**Intro to Java Development with Quarkus** 
**Time:** 15:15 - 16:00
**Speaker:** Kevin Dubois
**Abstract:** Java is a great programming language, however ‘traditional’ Java isn’t so great to work with when it comes to modern, Cloud Native development. Quarkus is a (fairly) new Java stack that addresses issues such as the typical slow startup time and rather large memory usage that hinder the adoption of Java in container and/or Serverless workloads. Quarkus is not just useful for optimizing resource usage though. There is also a big focus on improving the developer experience. In this session we’ll demonstrate how Quarkus is very easy and enjoyable to work with and allows developers to work with native GraalVM builds, containers and external dependencies such as databases, Kafka clusters, Kubernetes etc without being experts in any of these technologies (or even having to install them locally). After this session, the audience should come away with inspiration to build modern Cloud Native applications with Java, and have fun doing so!

**Comment by mabulgu** [comment]
Hi folks. In addition to the events above, I am going to present "Developing Kubernetes Operators with Java Operator SDK" and going to talk about the Quarkus extension + show a demo for it.

![Screenshot 2023-04-28 at 12 08 23](https://user-images.githubusercontent.com/10568159/235168332-3c2c83eb-6683-432c-aaa3-e1cedbe96c6d.png)

So this means we will have three sessions from Redhatters directly or indirectly related to Quarkus in just one day:)

**Reply by kdubois** [reply]
Awesome!  Looking forward to it!

**Reply by mabulgu** [reply]
> Awesome!  Looking forward to it!

Thanks Kevin! See you at the event!

**Reply by insectengine** [reply]
@mabulgu  - updated the list!

**Reply by mabulgu** [reply]
Oh, many thanks!
